modules/mobile/spec/workers/pre_cache_appointments_job_spec.rb

```diff
@@ -4,11 +4,11 @@
 require_relative '../support/iam_session_helper'
```

Comment:
The changes I made to these specs are difficult to read in the diff. I decided to remove some of the tests because I didn't think they were adding anything except complexity that made these tests needlessly difficult to refactor. They were testing feature flag and other appointment behaviors that are already tested in the appointments list request specs. I'm sure I added some of these tests myself, but at this time I'd rather keep them simple and do the more thorough testing in the integration (request) specs. It may be easier to read these specs by just looking at the file since the diff is so complex.